Emby.Server.Implementations/Library/ResolverHelper.cs

```diff
@@ -44,12 +45,14 @@ public static void SetInitialItemValues(BaseItem item, Folder? parent, ILibraryM
             var fileInfo = directoryService.GetFile(item.Path);
             if (fileInfo == null)
             {
-                throw new FileNotFoundException("Can't find item path.", item.Path);
+                return false;
```

Comment:
Needs some logging

Reply:
It's already logged at error level as not existing in the call to `directoryService.GetFile` [here](https://github.com/jellyfin/jellyfin/blob/d50c1b2d4bc0c85428671b288adef1b336da1156/Emby.Server.Implementations/IO/ManagedFileSystem.cs#L262), should I log another message on top of that?

Reply:
I suppose not